file_organizer_v2/docs/phase4/undo-redo.md

```
@@ -0,0 +1,827 @@
+python -m file_organizer.cli.undo_redo --list
+
+# Undo last operation
+python -m file_organizer.cli.undo_redo --undo
+
+# Redo last undone operation
+python -m file_organizer.cli.undo_redo --redo
+
+# Preview what would be undone (dry-run)
+python -m file_organizer.cli.undo_redo --undo --dry-run
+```
+
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)